src/steamship/data/file.py

```diff
@@ -117,59 +117,60 @@ def get(
     @staticmethod
     def create(
         client: Client,
-        filename: str = None,
```

Comment:
I decided to remove the `filename` arg as I perceive it as having little utility while adding some magic (deriving the filetype based on the filename).

Reply:
I'm good with this.  I think if we're going to have any automatic mappings from file types to plugins, it would be better to do them with mime types than with file extensions.